pytest_splunk_addon_ui_smartx/base_test.py

```diff
@@ -42,19 +39,13 @@ class SeleniumHelper:
     The helper class provides the Remote Browser
     """
 
-    sauce_username = None
-    sauce_access_key = None
-    sauce_tunnel_id = None
-    sauce_tunnel_parent = None
-    jenkins_build = None
-
     def __init__(
         self,
         browser,
         browser_version,
         splunk_web_url,
         splunk_mgmt_url,
-        debug=False,
+        local_run=False,
```

Comment:
This would require folks to update their code? I think it's better to avoid renaming variable there.

Reply:
Actually I didn't think this class is being used anywhere outside of smartx, but checked that and you are right. Going to update.